pdr_backend/trueval/subgraph.py

```diff
@@ -0,0 +1,100 @@
+from typing import List, Optional
+from pdr_backend.util.web3_config import Web3Config
+from pdr_backend.util.subgraph import query_subgraph, info_from_725
```

Comment:
This module needs a test, ie trueval/tests/test_subgraph.py. And of course if it gets renamed (as it should) then the test needs to reflect that too.

In general, when you're writing new functionality, please write the test at the same time. Test-first is faster, more reliable, and it means that someone else (or you) doesn't have to come back in a second time after-the fast.